tests/test_controls.py

```diff
@@ -329,25 +329,25 @@ def test_notify(self):
             {
                 "type": "request",
                 "uri": "ssap://system.notifications/createToast",
-                "payload": {"message": "test", "iconData": None, "iconExtension": "png"},
+                "payload": {"message": "test", "iconData": None, "iconExtension": None}
             }
         )
 
-    def test_notify__icon(self):
+    def test_notify_icon(self):
         def mock_get_icon_data(*args, **kwargs):
             return "dummy.png"
 
         pywebostv.controls.get_icon_data = mock_get_icon_data
 
         client = FakeClient()
         system = SystemControl(client)
-        system.notify("test", "myicon.png", block=False)
+        system.notify("test", icon_bytes="mydata", icon_ext="png", block=False)
```

Comment:
Just a tiny little callout: Once you fix the `postprocess=` param above, this would be of type `bytes`.